azure-functions/acbs-function/package.json

```diff
@@ -29,6 +29,7 @@
     "unit-test-ff": "jest --coverage --verbose --config=unit.ff.jest.config.js --passWithNoTests"
   },
   "dependencies": {
+    "@ukef/dtfs2-common": "1.0.0",
```

Comment:
Will this add a lot of bloat to the azure functions node_modules?

Reply:
Yes it will in a final docker build image.

Reply:
Are we still using libs common as it appears we're now using constants from the acbs folder?

it'd be quite nice to remove this, for micro benefits/spped gains when deploying. 

There's also the potential future issue regarding how the default libs common imports run at startup. As we are making use of `index` files, and a lot of constants that are evaluated at run time, we have to separate some imports, such as `mongo-db-client`, which would otherwise cause a failed startup. I'd like to avoid that issue being tied to the DF as this microservice should be really lean.